src/py42/clients/devices.py

```diff
@@ -53,37 +53,31 @@ def get_all(
         When no arguments are passed, all records are returned. To filter results, specify
         respective arguments. For example, to retrieve all active and blocked devices, pass
         active=true and blocked=true.
-
         `REST Documentation <https://console.us.code42.com/apidocviewer/#Computer-get>`__
 
         Args:
             active (bool, optional): Filters results by device state. When set to True, gets all
                 active devices. When set to False, gets all deactivated devices. When set to None
                 or excluded, gets all devices regardless of state. Defaults to None.
-            blocked (bool, optional): Filters results by blocked status, True or False. Defaults
+            blocked (bool, optional): Filters results by blocked status: True or False. Defaults
```

Comment:
Do we need to escape this colon? I've seen weird behaviors with using colons like this when building the docs.

Reply:
Looks good for me in local build.

Reply:
It's probably fine then! I just wanted to make sure.